en/docs/tutorials/integrating-with-solace.md

```diff
@@ -0,0 +1,144 @@
+# External Broker and Gateway Integration with API Manager
+
+WSO2 API Manager now supports integrating external third party Gateways and Brokers. Solace is an advanced event broker which is efficiently streaming events and information across cloud, on-premises, and IoT environments. In this section, you will integrate the Solace gateway environment with API Manager.
+
+## Integrating the Solace Event Broker
+
+1. Go to the [Solace PubHub+](https://solace.com/products/event-broker/) portal.
+2. Create a new account.
+   </br><a href="{{base_path}}/assets/img/tutorials/solace/create-new-account.png"><img src="{{base_path}}/assets/img/tutorials/solace/create-new-account.png" alt="Create new account" name="Create new account" width="600"></a>
+3. You will need the following information to integrate the broker with API Manager
+    1. Username
+    2. Password
+    3. Developer name
+    4. Organization name
+4. Open the `<API-M_HOME>/repository/conf/deployment.toml` file and add the following according to your account details. Make sure to restart the server after adding the configurations. 
+    ```toml
+        [[apim.gateway.environment]]
+        name = <Name of the Solace Environment>
+        display_name = <Display Name of the Solace Environment>
+        type = <Type of the Environment>
+        display_in_api_console = <Displays the environment under 'Try it' in the API Developer Portal, in the API console>
+        description = <Description about Solace Environment>
+        provider = <Vendor name of the environment. (Must be "solace")>
+        service_url = <Base URL of the Solace broker>
+        username = <Username of the Solace broker account>
+        password = <Password of the Solace broker account>
+        ws_endpoint = <WS Endpoint of the Solace broker environment>
+        wss_endpoint = <WSS Endpoint of the Solace broker environment>
+        http_endpoint = <HTTP Endpoint of the Solace broker environment>
+        https_endpoint = <HTTPS Endpoint of the Solace broker environment>
+        show_as_token_endpoint_url = false
+
+        [apim.gateway.environment.properties]
```

Comment:
@ruthryi Do we need to mention that these additional properties are not optional and those are mandatory?

Reply:
I think that info should go into the Config catalog. Can you pls provide which ones are mandatory?

Reply:
The gateway environment configs are already in the docs[1].

[1]https://apim.docs.wso2.com/en/latest/reference/config-catalog/#virtual-host-of-api-m-gateway-environment-configurations